tools/c7n_org/azuresubs.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2018 Capital One Services, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import yaml
+from azure.common.credentials import ServicePrincipalCredentials
+from azure.mgmt.resource.subscriptions import SubscriptionClient
+
+
+def main():
+    """
+    Generate a c7n-org subscriptions config file
+    """
+
+    tenant_auth_variables = [
+        'AZURE_TENANT_ID', 'AZURE_SUBSCRIPTION_ID',
+        'AZURE_CLIENT_ID', 'AZURE_CLIENT_SECRET'
+    ]
+
+    # Set credentials with environment variables if all
+    # required variables are present
+    if all(k in os.environ for k in tenant_auth_variables):
```

Comment:
at the moment the entire function body is behind this if condition, i'd suggest inverting the condition (ie `if not`), giving the user a warning message about the things your looking for and returning. Then the rest of the function body can be dedented.